test/invoice_test.rb

```diff
@@ -63,21 +61,14 @@ def test_delegates_merchant_to_parent_repo
   end
 
   def test_delegates_charge_to_parent_repo
-    parent.expect(:create_transaction_from, nil, [45, "4444333322221111", "10/13",  "success"])
-    invoice.charge(credit_card_number: "4444333322221111", credit_card_expiration: "10/13", result: "success")
+    parent.expect(:create_transaction_from,
+      nil,
+      [45, "4444333322221111", "10/13",  "success"]
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Unnecessary spacing detected.